src/styles/root.css

```diff
@@ -0,0 +1,35 @@
+@import url("./homePage.css");
+@import url("./quizOne.css");
+@import url("./rules.css");
+@import url("https://vision-ui-lib.netlify.app/styles/root.css");
+@import url("https://visionkart.netlify.app/styles/root.css");
+
+:root {
+	--color-bg: #1a202c;
+	--color-card-bg: #19212c;
+	--color-text: #f7f7f7;
+	--color-text-1: #d6d6d6;
+	--color-card-shadow: rgba(
```

Comment:
Try to use one constant color scheme either rgba or hex.

Reply:
I have used rgba for shadows only, I'll looks into this.

Reply:
I used rgba for shadow only. I'll look into these. Thank's Vineet for the detailed review.